### DISCUSSION TITLE
App Marketplace

### DISCUSSION THREAD

**Discussion by psteinroe** [supabase/supabase#9220] [Feature Requests]
Hey,

this is far out, but I think there is a huge opportunity for supabase to provide an app markeplace on top of the supabase platform where users can share (or sell) small apps that others can easily integrate in their own solution. These apps would be a bundle of a [postgres extension](https://github.com/supabase/supabase/discussions/9219), edge functions and maybe even ui code. An example for such an app would be a flatfile-like csv upload.

**Comment by thorwebdev** [comment]
Yes, this is a bit further out but something we're working towards with the Admin API. For now we have https://supabase.com/partners/integrations as a very first iteration of this.
